### ISSUE TITLE
How do I read the blendshape data?

### ISSUE THREAD

**Issue by vrisalab** [infosia/vmc2bvh#2]
I'm interested in getting the facial animations into blender or Unity, is there a way to do that? I guess I can write a script in blender or Unity that does the job but is the file meant to be used that way?

**Comment by infosia** [comment]
Right, currently vmc2bvh only saves blendshape data into `*.BLEND.json` and does not use it from program because there's no way to save blendshape info into bvh file. It is meant to be used by other script or something hopefully in the future.
